src/hermes/model/context.py

```diff
@@ -119,6 +120,14 @@ def error(self, ep: EntryPoint, error: Exception):
 
         self._errors.append((ep, error))
 
+    def purge_caches(self, *path: str) -> Path:
```

Comment:
path does not seem to be used, although I think it's good to make this method reusable.

Reply:
I guess the path should be remove here. If we want to selectively remove caches, the function should be called `purge_cache` or `remove_cache`.

Reply:
This functionality would be akin  to `git`'s `-C` option and should be implemented like it, which means it would set the hermes context's path. See https://github.com/hermes-hmc/workflow/pull/108#discussion_r1069185870 .